draft-bonaventure-quic-atsss-overview.mkd

```diff
@@ -232,9 +232,10 @@ to improve the up and download throughput performance of individual flows.
 The GiGA LTE and GiGA 5G services deployed using Multipath TCP in South Korea are examples of this use case {{IETFJ16}}.
 
 To support these different use cases in 5G networks, 3GPP is defining the
-Access Traffic Steering, Switching and Splitting (ATSSS) service {{TS23501}}. This work is
+Access Traffic Steering, Switching and Splitting (ATSSS) service {{TS23501}}
+to allow the simultaneous use of the combination of a 3GPP and a non-3GPP network interface. This work is
```

Comment:
I would rephrase this as either "simultaneous use" or "combination of", but I feel the two together are redundant.